.github/workflows/pull-request.yml

```diff
@@ -1,94 +0,0 @@
-name: Pull Request
-
-on:
-  push:
-    branches: '**'
-
-defaults:
-  run:
-    shell: bash
-
-jobs:
-  clippy:
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout Repository
-        uses: actions/checkout@v2
-
-      - name: Install Rust
-        uses: actions-rs/toolchain@v1
-        with:
-          toolchain: stable 
-          profile: minimal
-          components: clippy
- 
-      - name: Clippy
-        uses: actions-rs/cargo@v1
-        with:
-          command: clippy
-          args: --manifest-path rust/Cargo.toml
-
-  rustfmt:
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout Repository
-        uses: actions/checkout@v2
-
-      - name: Install Rust
-        uses: actions-rs/toolchain@v1
-        with:
-          toolchain: stable 
-          profile: minimal 
-      
-      - name: Check Format
-        uses: actions-rs/cargo@v1
-        with:
-          command: fmt
-          args: --manifest-path rust/Cargo.toml --all -- --check
-
-  build:
-    runs-on: ubuntu-latest
-    if: github.ref == 'refs/heads/master'
-    steps:
-      - name: Checkout Repository
-        uses: actions/checkout@v2 
-
-      - name: Install Rust
-        uses: actions-rs/toolchain@v1
-        with:
-          toolchain: stable 
-          profile: minimal
-
-      - name: Install Solana
-        run: curl -sSfL https://release.solana.com/stable/install | sh
-      
-      - name: Install Anchor Dependencies
-        run: sudo apt-get update && sudo apt-get upgrade && sudo apt-get install -y pkg-config build-essential libudev-dev
-      
-      - name: Install Anchor 0.24.2
-        run: npm i -g @project-serum/anchor-cli@0.24.2
-      
-      - name: Set Environment
-        run: echo "/home/runner/.cargo/bin" >> $GITHUB_PATH && echo "/home/runner/.local/share/solana/install/active_release/bin" >> $GITHUB_PATH
-
-      - name: Build Raindrops Programs
-        run: anchor build --skip-lint
-        working-directory: ./rust
-  
-  prettier:
```

Comment:
Why no prettier check?

Reply:
`yarn run lint:check` runs prettier in `js/cli` and `js/lib` workspaces.